src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="calendar calendar--start-day-tue">
```

Comment:
Read the task again and check if all the conditions are met.
(Add a modifier month-length for the calendar with values 28, 29, 30 and 31 (use @for))

Reply:
in my implementation you dont have to change it in html you can change css variable number-of-days to any number(<=31) and you will get layout for that many days

Reply:
It should still be in accordance with the task requirements, so please change that